CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added GCI-1442 - Python3.10 - Reduce memory footprint of dataclasses with slots
```

Comment:
Could you also add the URL of the PR and ID of the rule, like this example :

[#333](https://github.com/green-code-initiative/creedengo-rules-specifications/pull/333) Add JS variant of rule GCI505 - Idleness: Keep Screen On (addFlags)